server/pulp/server/webservices/views/serializers/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""
+Serialization package to transform Pulp's persistent resources from the database
+representation to a representation compatible with REST.
+
+This package supplies sub-modules, one per database collection. Each module
+supplies a number of methods to transform output for consumption by a REST
+client. The sub-modules are imported directly into this package to avoid
+namespace collisions. A developer only needs to import this package to use all
+the serialization modules.
+
+For example:
+
+from pulp.server.webservicesviews import serializeers
+bind_repr = serializers.bindings.serialize(bind)
+"""
+
+import binding
```

Comment:
I think we should not duplicate the structure of this module again here. Please delete lines 17 - 24. Python's normal import resolution will work well, will let us have one less thing to update (this file), and is the way the rest of the Pulp code works.